contrib/storage-kafka/src/test/java/org/apache/drill/exec/store/kafka/cluster/EmbeddedZKQuorum.java

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
Apache header should be in a form of comment, not Java doc. Please update here and in other newly added files.
Hope, somebody will add to check-style so we won't have to remind about it all the time.

Reply:
This is taken care.